vision.md

```diff
@@ -0,0 +1,41 @@
+# Buildfarmer Payload tasks process vision
+Subset of tasks defined to increase the quality of the ROS/Gazebo code by actively tracking and categorizing problems shipped with new code in a timely manner, making it easier for the development teams to:
+* Fix most errors when they are introduced.
+* Decide if an error that pops up in CI is a flaky test already existing in the codebase.
+* Identify how often an error occurs and decide how to proceed.
+
+## Internal process and rationale behind it
+
+We already have a priority document explaining the order of relevancy of the possible errors, this is a new set of standards to define what we expect of the internal buildfarmer process:
+
+1. Auto-log any error appearing in the nightlies, even when there is no one around. This could serve future investigations.
+2. Any person can be assigned to Buildfarmer Payload in case of necessity, this implies:
```

Comment:
I haven't read the rest of the documents in here, but the "Buildfarmer Payload" role isn't defined prior to its use here.  I think we either need to define what it is first, or link to a document that does so.

In particular, it is unclear to me where the person assigned to the Buildfarm Payload role will come from.

Reply:
> the "Buildfarmer Payload" role isn't defined prior to its use here

The first paragraph is supposed to explain what's the Buildfarm Payload role, I'll add a bit more to make it explicit.

Reply:
> it is unclear to me where the person assigned to the Buildfarm Payload role will come from

This was deliberately loose, as of today I could state that this role will be assigned to someone from the infra team. 

However, our vision for the role is having all the tools available publicly and make it possible for anyone from the community to run take this role (if someone ever wanted to). I'll add a paragraph to track this.